src/ontology/mondo-ingest.Makefile

```diff
@@ -593,11 +593,34 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
-# General
-update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
 
 EXTERNAL_CONTENT_DIR=external
 
+EXTERNAL_FILES = efo-proxy-merges \
+		mondo-clingen \
+		mondo-efo \
+		mondo-medgen \
+		mondo-omim-genes \
+		mondo-otar-subset \
+		nando-mappings \
+		nord \
+		ordo-subsets
+
+tmp/mondo-incl-external.owl: $(foreach n,$(EXTERNAL_FILES), $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl) mondo.owl
+	$(ROBOT) merge $(foreach n,$^, -i $(n)) \
+		filter --term MONDO:0000001 --term MONDO:0021125 --term MONDO:0042489 --term MONDO:0021178 --select "annotations self descendants" --signature true -o $@
+
+tmp/mondo-incl-robot-report.tsv: tmp/mondo-incl-external.owl config/robot_report_external_content.txt
+	$(ROBOT) report -i $< --profile config/robot_report_external_content.txt --fail-on None -o $@
+
+$(EXTERNAL_CONTENT_DIR)/processed-%.robot.tsv: tmp/mondo-incl-robot-report.tsv
+	python $(SCRIPTSDIR)/post_process_externally_managed_content.py --emc-id $* --emc-dir $(EXTERNAL_CONTENT_DIR) --robot-report $<
```

Comment:
Design `nit`: In other pipelines, we've frowned on the method of a CLI where we just pass the ID of a source, and have preferred to list all of the source-related inputs and outputs as individual params.

I don't lean as strongly towards this preference though, and I understand why @matentzn went with this approach instead as it is much quicker to implement / maintain.

Reply:
You are right. Thanks for pointing that out. I changed the script to take only one TSV file as input, and write 1 tsv file as output.

Reply:
(I like being very explicit on IO parameters in pipelines)